frontend/src/pages/pipelines/detail/PipelineDetail.tsx

```diff
@@ -66,8 +66,10 @@ const PipelineDetail: React.FC = () => {
         setLoadingData(false);
         setLoadingPipeline(false);
       }
-    } catch (error) {
-      setLoadingPipeline(false);
+    } catch (error: any) {
+      if (error?.message?.response?.status === 404) {
+        window.location.href = '/error/404';
```

Comment:
Is the url path `/error/404` a convention in React?

Reply:
There is no such convention in React.